src/test/java/io/github/wimdeblauwe/hsbt/thymeleaf/HtmxDialectTest.java

```diff
@@ -114,4 +114,14 @@ void testWithThEach() throws Exception {
                 .containsPattern("hx-target-div-with-th-each.*hx-target=\"2\"")
                 .containsPattern("hx-target-div-with-th-each.*hx-target=\"3\"");
     }
+
+    @Test
+    void testWithHxVals() throws Exception {
```

Comment:
Maybe also add a test with multiple values, maybe with a string, a boolean and a number to cover those 3 options?